guides/source/action_view_helpers.md

```diff
@@ -125,6 +144,15 @@ stylesheet_url "application"
 # => http://www.example.com/assets/application.css
 ```
 
+#### video_tag
+
+Generate an HTML video tag with source(s), either as a single tag for a string source or nested source tags within an array for multiple sources.
```

Comment:
I think it's also worth noting that we can pass paths, files and ActiveStorage objects here too

Reply:
Basically these all are abstract, if they want to get more details, they shoulda look into the api documentation.